tools/firtool/firtool.cpp

```diff
@@ -710,6 +710,7 @@ int main(int argc, char **argv) {
     firrtl::registerPasses();
     om::registerPasses();
     sv::registerPasses();
+    hw::registerPasses();
```

Comment:
This pulls in a few passes more than strictly needed, including some passes unreachable from `firtool`.

I'm okay with this but as HW grows more passes we might not want to pull them all in uncritically.

Looks like we're already pulling in all SV passes which is more problematic, so something to cleanup ....